CoinDock-app/src/Shared/Date/DatePick.js

```diff
@@ -15,10 +15,10 @@ export const dateValidation = (value) => {
   }
   return error;
 };
-function DatePick({ name, onChange, formErrors }) {
-  const [selectedDate, setSelectedDate] = useState("");
-  const [fieldsTouched, setFieldsTouched] = useState(false);
-
+function DatePick({ name, onChange, formErrors,value}) {
+  
+ const [selectedDate, setSelectedDate] = useState(new Date(value));
```

Comment:
Initially some times I thinks there should be no value in the field. 
If we set the selectedDate with new Date(value) In those scenarios even those please default value will be shown. I think mostly current date.
So, I think we should set when there is value in the field or not.

Reply:
Resolved